datafusion/sqllogictest/test_files/array.slt

```diff
@@ -2752,7 +2752,7 @@ select range(5),
        range(1, -5, 1)
 ;
```

Comment:
```suggestion
       range(1, -5, -1)
;
```

Reply:
move the test to the sqllogistest, overall LGTM 👍

Reply:
Updated~